proto/osmosis/txfees/v1beta1/query.proto

```diff
@@ -17,6 +17,12 @@ service Query {
     option (google.api.http).get = "/osmosis/txfees/v1beta1/fee_tokens";
   }
 
+  rpc TxFeesByDenom(QueryTxFeesByDenomRequest)
```

Comment:
This name doesn't sound right. Its returning the spot price

Reply:
Hmm, yeah agreed how abt `SpotPrice`?

Reply:
Oh yeah, these should be IBC denoms / that will cause issues

Reply:
Perhaps `TxFeeDenomSpotPrice`? (Or I guess the tx fee part is implied by the route including `txfees`)